models/evc.py

```diff
@@ -1261,7 +1335,7 @@ def _send_flow_mods(dpid, flow_mods, command='flows', force=False):
             force(bool): True to send via consistency check in case of errors
 
         """
-
+        #raise FlowModException(str(1234))
```

Comment:
Leftover prototyped/exploratory work here. Not good.

Reply:
Deleted